.github/actions/kind/action.yml

```diff
@@ -47,3 +47,12 @@ runs:
         echo "KinD cluster:"
         kubectl cluster-info
         kubectl describe nodes
+
+    - name: Install Ingress controller
+      shell: bash
+      run: |
+        VERSION=controller-v1.6.4
```

Comment:
Should we move this variable to something that's definied at the base level of the Makefile, and then pass it down to test/e2e/kind.sh as well?

Reply:
I think that's more an testing internal detail, than something that should be parameterised as for the core dependencies.